docs/getting-started.md

```diff
@@ -146,7 +151,7 @@ For everything else the cloud builder GUI is self-explanatory. Hence, no need fo
 1. Select an LFS size, 64KB is likely going to be large enough
 1. Select other options and build
 
-[↑ back to matrix](#task-os-selector)
+[↑ back to matrix](#task--os-selector)
```

Comment:
That's possibly fixing a non-existent bug, see https://nodemcu.readthedocs.io/en/latest/getting-started/#task-os-selector

-> `#task-os-selector`

In case GitHub and RTD use different to-anchor algorithms (caused by the '/' ?) then maybe we need to adjust the title.

Reply:
Oops, should have checked at RTD. changed the title.

Reply:
Btw: is there a way to check RTD before at least having it on dev branch?
Btw2: Have you seen the PR number?

Reply:
> Btw: is there a way to check RTD before at least having it on dev branch?

Two options:
- Create a RTD account for yourself and create a project pointing at your forked NodeMCU repo.
- Install [MkDocs](https://github.com/mkdocs/mkdocs/) locally and run `mkdocs serve` in the NodeMCU root (ignoring the warnings about invalid source code references). Question is: which MkDocs version? I lost track.

> Btw2: Have you seen the PR number?

😄 not until you mentioned it, congrats for pulling a repdigit ticket number